bazelrio/scripts/deploy/Deploy.java

```diff
@@ -108,16 +124,20 @@ public static void main(String[] args) throws IOException {
         // Copy dynamic libraries
         for (String dynamicLibraryPath : dynamicLibraryPaths) {
             File dynamicLibrary = runfile(runfiles, dynamicLibraryPath);
-            progressBar.setExtraMessage(dynamicLibrary.getName());
+            progressBar.setExtraMessage("Deploying " + dynamicLibrary.getName());
             String dynamicLibraryDestination = String.format("/usr/local/frc/third-party/lib/%s", dynamicLibrary.getName());
             scp.copy(new FileSystemFile(dynamicLibrary), dynamicLibraryDestination);
             progressBar.step();
         }
+	    progressBar.close();
 
         // Restart robot code
+	    System.out.print("Restarting robot code... ");
         runCommand(client, "sync", verbose);
         runCommand(client, "ldconfig", verbose);
         runCommand(client, ". /etc/profile.d/natinst-path.sh; /usr/local/frc/bin/frcKillRobot.sh -t -r", verbose);
+        System.out.println("Done.");
+        System.out.print("Deploy completed!");
```

Comment:
Super nit, seems redundant.

Reply:
You mean the prints sjould be merged?
The 'Done' is for restarting robot code.

I split them because then another step can be added in between without refactoring